nixos/modules/services/torrent/transmission.nix

```diff
@@ -66,21 +67,15 @@ in
       };
 
       downloadDirPermissions = mkOption {
-        type = types.str;
+        type = types.nullOr types.str;
         default = "770";
         example = "775";
         description = ''
           The permissions to set for download-dir and incomplete-dir.
-          They will be applied on every service start.
+          If not null, they will be applied on every service start.
```

Comment:
FYI - after a quick glance over I thought I would mention that `tmpfiles` are not applied every time services start. Maybe I missed something, but the description makes it sound like you're depending on that.

Reply:
This remark is correct and I think the currently suggested description is better.